### PR TITLE
Feature/download/cica 237

### PR DESCRIPTION
**Pull request by neil-stephen-mcgonigle**
- added download functionality

### CODE REVIEW

File: test/download-transformer.test.js [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+                                content: schemaArrayOfThemesContent,
+                                pageTitle: 'Check your answers'
+                            };
+
+                            expect(removeIndentation(result)).toEqual(removeIndentation(expected));
+                        });
+                    });
+                });
+
+                describe('And the summaryStructure key is present in summaryInfo', () => {
```
Comment (armoj) [inline test/download-transformer.test.js:86]: duplicate describe block as line 59

File: test/download-transformer.test.js [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+                                '{% from "summary-list/macro.njk" import govukSummaryList %}'
+                            ],
+                            id: 'p--check-your-answers'
+                        };
+
+                        expect(removeIndentation(result)).toEqual(removeIndentation(expected));
+                    });
+                });
+
+                it('should return a govukSummaryList in the order defined by the template and append answers which are supplied in the body but not defined in the template with no change links', () => {
```
Comment (armoj) [inline test/download-transformer.test.js:118]: any way you could rewrite this test name?

**Review by armoj — CHANGES_REQUESTED** [review]

File: test/download-transformer.test.js (hunk 2 above)
Comment (neil-stephen-mcgonigle) [reply test/download-transformer.test.js]: This is a modification of this [test ](https://github.com/CriminalInjuriesCompensationAuthority/q-transformer/blob/b02d10501058f336c654882d484b1611954b3b81/test/q-transformer.test.js#L1696)for the download test

**Comment by neil-stephen-mcgonigle** [comment]
Proposed version bump minor

**Review by armoj — APPROVED** [review]

**Pull request merged**